cfgov/jinja2/v1/_includes/organisms/reg-comment.html

```diff
@@ -34,7 +38,12 @@
             If you would like to include an attachment with your response,
             please submit your comment at
             <a class="icon-link icon-link__external-link"
-               href="https://www.regulations.gov/">
+               href="https://www.regulations.gov/{{
+                    '#!submitComment;D=' + value.document_id
```

Comment:
Yes, if `generic_regs_link` is false, the link should just be https://www.regulations.gov/